projects/core/directives/dropdown/dropdown-position.directive.ts

```diff
@@ -20,7 +20,7 @@ export class TuiDropdownPositionDirective extends TuiPositionAccessor {
 
     readonly type = 'dropdown';
     readonly accessor = tuiFallbackRectAccessor('dropdown')(
-        inject(TuiRectAccessor) as any as readonly TuiRectAccessor[],
+        inject<any>(TuiRectAccessor) as readonly TuiRectAccessor[],
```

Comment:
Not sure `as readonly TuiRectAccessor[]` adds anything here

Reply:
fixed